ztp/source-crs/StoragePV.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: pv-storage
+  annotations:
+    ran.openshift.io/ztp-deploy-wave: "10"
+spec:
+  storageClassName: example-storage-class
+  capacity:
+    storage: 10Mi
+  volumeMode: Filesystem
+  accessModes:
+  - ReadWriteOnce
+  persistentVolumeReclaimPolicy: Delete
+  local:
+    path: /mnt/local-storage
+  nodeAffinity:
+    required:
+      nodeSelectorTerms:
+      - matchExpressions:
+        - key: node-role.kubernetes.io/worker
```

Comment:
should this be /worker or /$mcp to align with the following config?

https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/source-crs/PtpOperatorConfigForEvent.yaml#L10

Reply:
updated